src/rf/apps/core/models.py

```diff
@@ -274,6 +278,12 @@ class LayerImage(Model):
         max_length=255,
         help_text='Name of S3 bucket'
     )
+    source_s3_bucket_key = CharField(
+        blank=False,
+        default='',
```

Comment:
Since blank is `False` you are forced to enter junk data from the admin interface to save or update this.